projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -4,43 +4,30 @@
 
 ## 11.0-a.7 - 2022-05-18
 ### Enhancements
-- Added jetpack v4 publicize end-point. [#24293]
-- Admin Menu: Refactor upsell nudge to be async [#24304]
-- Email subscriptions: Update the default body of the "Confirmation request" email [#24389]
-- Moved SocialServiceIcon component from Jetpack Icons.js file to js-package/components. Updated it's ref in the Jetpack plugin directory [#23795]
-- Updated Jetpack Scan feature list. [#23863]
-- We now only perform one network request when utilizing any number of premium content and or payment blocks. Refactored the product memberships store to reduce complexity. [#24333]
+- Email subscriptions: update the default body of the "Confirmation request" email. [#24389]
+- Membership Products Store: refactor the product memberships store to reduce complexity and improve speed. [#24333]
+- VideoPress: add support for automatic seekbar color. [#24330]
 
 ### Improved compatibility
-- JSON API: update the theme endpoint with information concerning pending updates. [#24392]
-- Update usage of the connection package by Jetpack [#24309]
-- VideoPress: add useAverageColor to wpvideo shortcode to support automatic seekbar color. [#24330]
+- General: improve the connection sharing between Jetpack and standalone plugins. [#24309]
 
 ### Bug fixes
-- Add activation/deactivation hooks for proper setup of the waf module [#24153]
-- Changed the logic in should_submit_post_pre_checks to allow publiscizing via ajax requests. [#24387]
-- Make sure payment blocks only attempt to auto-select a product if it exists. [#24407]
-- Remove search widget CTA on uneligible themes [#24386]
+- WAF: add activation/deactivation hooks for waf module. [#24153]
```

Comment:
```suggestion
- WAF: add activation/deactivation hooks for WAF feature. [#24153]
```

Reply:
Given that this is still being developed, maybe this should be in "Other" ?

Reply:
Last time we decided to keep WAF in the Enhancements section but I'm not sure what was the reasoning behind it.